lib/cAst.mli

```diff
@@ -0,0 +1,22 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+(** The ast type contains generic metadata for AST nodes *)
```

Comment:
Wouldn't it make sense to expand the "generic metadata for AST nodes" here? That is to describe more precisely what you mean? E.g. "data shared by all AST nodes" ?

I get a feeling that words like "generic" and "metadata" are frequently abused.